Controller/Transaction/CommitWebpay.php

```diff
@@ -78,7 +80,7 @@ public function execute()
             $this->log->logInfo('Request: payload -> ' . json_encode($request));
 
             return $this->handleRequest($request);
-        } catch (Exception $exception) {
+        } catch (MissingArgumentException | TransactionCommitException $exception) {
```

Comment:
`TransactionCommitException` is catched in `commitTransaction` method, is unnecessary catch here.
commit method from transbank-sdk can throw `GuzzleException`